index.html

```diff
@@ -139,7 +139,75 @@ <h2>What is a Hackathon?</h2>
+                    <div class="panel">
+                        <p>
+                            Anyone currently enrolled as a student can attend! If you don't fit that description, you're
+                            absolutely welcome to attend as a mentor or volunteer. All attendees must be 18 years or older.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <button class="accordion"><i class="fa fa-plus fa-plus-red" aria-hidden="true"></i>Do RIT students need to apply?</button>
+                    <div class="panel">
+                        <p>
+                            Anyone currently enrolled as a student can attend! If you don't fit that description, you're
+                            absolutely welcome to attend as a mentor or volunteer. All attendees must be 18 years or older.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <button class="accordion"><i class="fa fa-plus fa-plus-red" aria-hidden="true"></i>When will I know if I got in?</button>
+                    <div class="panel">
+                        <p>
+                            Anyone currently enrolled as a student can attend! If you don't fit that description, you're
+                            absolutely welcome to attend as a mentor or volunteer. All attendees must be 18 years or older.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <button class="accordion"><i class="fa fa-plus fa-plus-orange" aria-hidden="true"></i>What should I bring?</button>
+                    <div class="panel">
+                        <p>
+                            Anyone currently enrolled as a student can attend! If you don't fit that description, you're
+                            absolutely welcome to attend as a mentor or volunteer. All attendees must be 18 years or older.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <button class="accordion"><i class="fa fa-plus fa-plus-orange" aria-hidden="true"></i>Should I bring my own bricks...</button>
+                    <div class="panel">
+                        <p>
+                            Anyone currently enrolled as a student can attend! If you don't fit that description, you're
+                            absolutely welcome to attend as a mentor or volunteer. All attendees must be 18 years or older.
+                        </p>
+                    </div>
+                </div>
+            </div>
+
+            <p id="contact">Don't see your question here? <a href="#" rel="noopener" target="_blank">Contact Us</a></p>
```

Comment:
Ah, the notorious `rel="noopener"`. Is it required on every `target="_blank"`?

Reply:
I generally use it by default for security reasons but I think it would be fine to not use it for a static site. Do you have a preference?

Reply:
Let's not use it quite yet as it's not on all links, and it'll be easier to add them all at once after we get the initial version (most PRs) ready.